website/docs/r/container_node_pool.html.markdown

```diff
@@ -14,19 +14,19 @@ and
 [API](https://cloud.google.com/container-engine/reference/rest/v1/projects.zones.clusters.nodePools).
 
 ## Example usage
-
+### Standard usage
 ```hcl
 resource "google_container_node_pool" "np" {
   name               = "my-node-pool"
   zone               = "us-central1-a"
   cluster            = "${google_container_cluster.primary.name}"
-  initial_node_count = 3
+  node_count = 3
```

Comment:
The spacing on this is a bit off now- the lines above should have the `=` aligned with this one (same comment for the other examples too)

Reply:
Done